### DISCUSSION TITLE
smart auto clicker shut down on bluestack X

### DISCUSSION THREAD

**Discussion by noobiessss** [Nain57/Smart-AutoClicker#433] [Q&A]
smart auto clicker works for a minute then closes and it only happens on bluestacks (windows 11). works fine on other emulator. bluestacks and other emulator is running the same game.

**Comment by Cyber5imon** [comment]
Please try the latest [Beta 3.00v4](https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.0-beta04) version.

**Comment by noobiessss** [comment]
i have. same result

**Comment by nicocposs** [comment]
Still the same issue here, using the latest 3.0.6 release.
